.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,2 @@
+Please see our contribution guide at
+[dvc.org](https://dvc.org/doc/user-guide/contributing-documentation#contributing-to-the-documentation).
```

Comment:
do we actually need the #contributing-to-the-documentation part?

Reply:
@shcheklein Not necessarily. This just scrolls down the page a little bit so it starts exactly from the heading.

Reply:
@algomaster99 let's remove it? just to make sure that we always go the very top of the page.

Reply:
@shcheklein One more thing. The guide in [dvc](https://github.com/iterative/dvc) has the content written in `h3` font. Shall I change it here too to maintain uniformity?

Reply:
I don't have a strong preference on this, to be honest :) Up to you.